### ISSUE TITLE
Hosted Fields hostedFieldsInstance.focus() not bringing up credit card input keyboard (mobile safari / mobile chrome)

### ISSUE THREAD

**Issue by zaklampert** [braintree/braintree-web#319]
### General information

* SDK version: 3.22.0
* Environment:  Sandbox
* Browser and OS: Android Chrome (newest) iOS safari (newest)

### Issue description

Calling the `.focus()` method does not automatically bring up the credit card input keyboard for iOS or Android. On Android Chrome, the field shows the cursor as if it's focused, but does not bring up the keyboard, requiring an additional tap on the field.

Here's my code, which produces no errors.
```
hostedFieldsInstance.focus('number', (err) => {
        err && console.log(err);
      });
```

**Comment by lilaconlee** [comment]
For iOS Safari, this is a known behavior. There are known issues showing the iOS Safari keyboard with JavaScript. We'll add some documentation around this.

For Android Chrome, we've been unable to replicate this behavior. Android keyboard behavior varies between devices, so if you could send what phones you've seen this behavior on we can debug further.

**Comment by lilaconlee** [comment]
Closing this issue as we've been unable to reproduce the buggy Android behavior, but let us know if you have more details and we can re-open the issue!

**Comment by andismith** [comment]
@lilaconlee 

> For iOS Safari, this is a known behavior. There are known issues showing the iOS Safari keyboard with JavaScript. We'll add some documentation around this.

Please can you point me to where this documentation is? Thanks

**Comment by jackellenberger** [comment]
Sorry @andismith, we dropped the ball on updating the docs- they will be updated with the next release.

http://braintree.github.io/braintree-web/current/HostedFields.html#focus-examples will now read: 
```
// In Firefox, the focus method can be suppressed
//   if the element has a tabindex property or the element
//   is an anchor link with an href property.
// In Mobile Safari, the focus method is unable to
//   programatically open the keyboard, as only
//   touch events are allowed to do so.
```

**Comment by crookedneighbor** [comment]
FYI, we just released some changes in v3.78.2 that addresses the quirks in iOS Safari around focussing programmatically.
